cmake/README.md

```diff
@@ -42,3 +42,35 @@ The following targets are available for build system after configuration:
 
 Sudirectories are added as labels for each test and can be used with `ctest -L <label>`.
 For example, `<root>/test/path/to/test.pass.cpp` will have `path` and `to` labels.
+
+## How to use oneDPL package from CMake
+
+oneDPLConfig.cmake and oneDPLConfigVersion.cmake are included into oneDPL distribution.
+
+These files allow to integrate oneDPL into user project with the [find_package](https://cmake.org/cmake/help/latest/command/find_package.html) command. Successful invocation of `find_package(oneDPL <options>)` creates imported target `oneDPL` that can be passed to the [target_link_libraries](https://cmake.org/cmake/help/latest/command/target_link_libraries.html) command.
+
+For example:
+
+```cmake
+project(Foo)
+add_executable(foo foo.cpp)
+
+# Search for oneDPL
+find_package(oneDPL REQUIRED)
+
+# Connect oneDPL to foo
+target_link_libraries(foo oneDPL)
+```
+
+Availability of DPC++ and oneTBB backends is automatically checked during the invocation of `find_package(oneDPL <options>)`:
+
+- macro `ONEDPL_USE_TBB_BACKEND` is set to `0` if oneTBB is not available;
+- macro `ONEDPL_USE_DPCPP_BACKEND` is set to `0` if DPC++ is not available.
```

Comment:
Also, we can mention about the enabling back-ends.
Here we can say that if these macros are undefined or not equal to 0, the use of corresponding back-end is supposed (enable to use).

Reply:
I put the reference to documentation and provided a simple example, please take a look